src/commands/handlers/registration.js

```diff
@@ -22,7 +22,7 @@ const handlers = {
         // to an IRC server, not now(). Send a PING so that we can get a reliable time from PONG
         if (handler.network.cap.isEnabled('server-time')) {
             // Ping to try get a server-time in its response as soon as possible
-            handler.connection.write('PING ' + Date.now());
+            handler.connection.write('PING kiwitime-' + Date.now());
```

Comment:
Any reason for the prefix?

Reply:
I added the prefix purely to be consistent with https://github.com/kiwiirc/irc-framework/blob/b3dc36fd4e46a8c33e1c4bdf39b2d3c1c9ebcb95/src/client.js#L382